protected/humhub/modules/user/models/Invite.php

```diff
@@ -90,7 +86,7 @@ public function beforeSave($insert)
         if ($insert && $this->token == '') {
             $this->token = Yii::$app->security->generateRandomString(self::TOKEN_LENGTH);
         }
-
+        $this->setMeta($insert, $this->user_originator_id);
```

Comment:
Need to throw false in beforeSave() If created_by or updated_by will set to NULL(user_id is wrong) then?

Reply:
For example:

```php
public function beforeSave($insert)
    {
        if (!parent::beforeSave($insert)) {
            return false;
        }
        if ($insert && $this->token == '') {
            $this->token = Yii::$app->security->generateRandomString(self::TOKEN_LENGTH);
        }
        return $this->setMeta($insert, $this->user_originator_id);
    }
```